pkg/vgmanager/lvm_test.go

```diff
@@ -0,0 +1,69 @@
+package vgmanager
+
+import (
+	"bytes"
+	"os/exec"
+	"testing"
+)
+
+// TODO: unit test creation of Volume Groups and replace bash script
+
+var fileName = "vg-pool-"
+var vgName = "test-vg"
+
+func setup(name string) string { //nolint:golint,unused
```

Comment:
is the test creating a new VG and deleting that? 
If yes, then do we really need to do that?

Reply:
1. Currently this test will not work in CI as `nsenter` doesn't work and being skipped, need to refactor that to be testable
2. The intention is to create a volume group manually (eventually do that via vgmanager functions) and remove that via function being used in this PR, incase if deletion fails teardown will remove it quietly